CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed bugs
 
+* Commands accepting filesets hint to look at the documentation in presence of
+  a syntax error.
```

Comment:
nit: delete changelog entry? I don't think users have to know the error message is improved.